src/CommunityToolkit.Maui.Core/Primitives/RatingViewShape.cs

```diff
@@ -0,0 +1,41 @@
+﻿namespace CommunityToolkit.Maui.Core.Primitives;
```

Comment:
Please fix the namespace here once we've decided how to reconcile `public enum RatingViewShape` + `public sealed class RatingViewShape` 

```suggestion
﻿namespace CommunityToolkit.Maui.Core;
```